tests/integration/13.DFWGlobalOptions.Tests.ps1

```diff
@@ -33,15 +33,15 @@ Describe "DFW Global Properties" {
         import-module $pnsxmodule
         $script:DefaultNsxConnection = Connect-NsxServer -vCenterServer $PNSXTestVC -Credential $PNSXTestDefViCred -ViWarningAction "Ignore"
         # Threshold Variables
-        $script:cpuThreshold = "75"
-        $script:cpuThreshold1 = "85"
-        $script:memoryThreshold = "75"
-        $script:memoryThreshold1 = "85"
-        $script:cpsThreshold = "125000"
-        $script:cpsThreshold1 = "200000"
-        $script:cpsDefault = "100000"
-        $script:memorydefault = "100"
-        $script:cpudefault = "100"
+        $script:cpuThreshold = ("75").ToString()
```

Comment:
"75" is already a string as its quoted (use .gettype() to check).  

In general if you want to be explicit - specify the type when declaring a var - ie [string]$script:cpuThreshold = "75" 

In this case, Im pretty sure you will find the changes in the test are not required.  Confirm?